demo-app/src/LocationsMap.jsx

```diff
@@ -19,7 +27,14 @@ function LocationsMap() {
   return (
     <Box ref={containerRef} flex background="background-contrast">
       {geolocation && (
-        <Map id="map" ref={mapContainerRef} center={geolocation} zoom={6}>
+        <Map
+          id="map"
+          ref={mapContainerRef}
+          center={LocationBounds({ locations: geojsonLocations }).getCenter()}
```

Comment:
Let's leave this as is for now:

```suggestion
          center={geolocation}
```